tests/acceptance/features/recursion_error.feature

```diff
@@ -0,0 +1,1090 @@
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
+      | unused |
```

Comment:
Maybe explain in the feature or scenario title why you have chosen exactly 1084 scenario values.

(Note: This is probably a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)), so it requires some explanation.)